library/src/main/java/com/novoda/downloadmanager/lib/DownloadClientReadyChecker.java

```diff
@@ -3,6 +3,9 @@
 public interface DownloadClientReadyChecker {
     Ready READY = new Ready();
 
+    /**
+     * Warning: This method is executed on a background thread
```

Comment:
no need to say "warning" it's just executed on a background thread, people will probably we happy rather than warned :)